driver/normalizer/normalizer.go

```diff
@@ -90,26 +103,34 @@ var Normalizers = []Mapping{
 
 	MapSemantic("CPPASTImplicitName", uast.Identifier{}, MapObj(
 		Obj{
-			"Name": Var("name"),
+			"Name":                 Var("name"),
+			"IsQualified":          Var("ignIsQual"),
+			"IsAlternate":          Var("ignIsAlternate"),
+			"IsOverloadedOperator": Var("ignIsOver"),
 		},
 		Obj{
 			"Name": Var("name"),
 		},
 	)),
 
-	MapSemantic("CPPASTQualifiedNames", uast.QualifiedIdentifier{}, MapObj(
-		Obj{
-			"Prop_AllSegments": Var("names"),
-		},
-		Obj{
-			"Names": Var("names"),
-		},
-	)),
+	// Disabled: parts can be anything, not only identifiers (like function calls or object instantiations)
+	// and the SDK always expects identifiers (but should except Any[] for this object)
```

Comment:
As discussed in Slack, `QualifiedIdentifier` in SDK works as intended. The one in C++ needs to be split into `QualifiedIdentifier` and "selector expressions" similar to [this](https://github.com/bblfsh/go-driver/blob/master/fixtures/calls.go.native#L904).

Reply:
I've noticed that all the parts of a `QualifiedName` in the native AST implement the interface `IASTName` so with a somewhat huge `Case` it should be possible to get all the name parts (working on it right now).

Reply:
Fixed in next commit, but this creates the new problem that `FunctionGroup.Name` expect an Identifier and sometimes the value is a `QualifiedIdentifier` so I'll need to join the `QualifiedIdentifier.Names` into a single string to pass as a single `Identifier` (if there is a better solution please tell me).

Reply:
Hmm, you are right, I haven't expected an `Alias` to be a `QualifiedIdentifier`. I'll check it.